test/samples/solidity/latest_08.sol

```diff
@@ -35,3 +35,23 @@ contract UsesNewAddressMembers {
         bytes32 codeHash = address(0).codehash;
     }
 }
+
+contract CatchPanic {
+    function test() public {
+        UsesNewAddressMembers c = new UsesNewAddressMembers();
+
+        try c.test() {
+            
+        } catch Error(string memory reason) {
+            revert(reason);
+        } catch Panic(uint _code) {
```

Comment:
Where is `Panic` defined? I thought in 0.8.1 you had to define the error types?

Reply:
`Panic` is indicated as velur of `errorName` property of `TryCatchClause` node: https://github.com/ConsenSys/solc-typed-ast/blob/master/src/ast/implementation/statement/try_catch_clause.ts#L10